src/components/Widget.tsx

```diff
@@ -59,6 +58,11 @@ const slideIn = keyframes`
     transform: translateX(calc(100% - 0.25em));
   }
 `
+const slideBack = keyframes`
```

Comment:
would consider renaming `slideBack` or `slideOut` since they don't feel like opposites. maybe `slideIn` and `slideOut`. or `slideRight` and `slideLeft`?

Reply:
`slideRight` can mean sliding in or out, but `slideInLeft` conveys the entire action.
Note that I'm naming the action, and leaving it to the CSS (not the keyframes) to assign semantic meaning.